x/data/module/module.go

```diff
@@ -0,0 +1,97 @@
+package module
+
+import (
+	"encoding/json"
+	"github.com/cosmos/cosmos-sdk/x/evidence/types"
+	"github.com/regen-network/regen-ledger/x/data/server"
+
+	"github.com/gorilla/mux"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	sdkclient "github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	"github.com/regen-network/regen-ledger/x/data"
+	"github.com/regen-network/regen-ledger/x/data/client"
+)
+
+const (
+	StoreKey = types.ModuleName
```

Comment:
```suggestion
	StoreKey = data.StoreKey
```
We set this constant in the `x/data` package, shouldn't we pull from there? As implemented this pulls the evidence module's `ModuleName` from `x/evidence/types`

Reply:
Weird